engine/classes/Elgg/Application.php

```diff
@@ -46,6 +46,7 @@ class Application {
 	 * @var string[]
 	 */
 	private static $public_services = [
+		'assets' => true,
```

Comment:
Thanks for keeping me honest! Will get to those things asap...

On Mon, Jun 8, 2015, 6:16 PM Steve Clay <notifications@github.com> wrote:

> In engine/classes/Elgg/Application.php
> <https://github.com/Elgg/Elgg/pull/8434#discussion_r31975977>:
>
> > @@ -46,6 +46,7 @@ class Application {
> >  	 * @var string[]
> >  	 */
> >  	private static $public_services = [
> > +		'assets' => true,
>
> Also needs property-read for the interface, a section in
> guides/services.rst, and clearer guide to usage in the interface.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8434/files#r31975977>.
>